pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -97,15 +95,19 @@ func TestBinderNew(t *testing.T) {
 		assert.True(t, found)
 		assert.Len(t, containers, 1)
 
-		c := corev1.Container{}
-		u := containers[0].(map[string]interface{})
-		err = runtime.DefaultUnstructuredConverter.FromUnstructured(u, &c)
-		assert.NoError(t, err)
-		assert.NotEmpty(t, c.Env[0].Value)
-
-		parsedTime, err := time.Parse(time.RFC3339, c.Env[0].Value)
-		assert.NoError(t, err)
-		assert.True(t, parsedTime.Before(time.Now()))
+		// I believe the test subject here is the environment variable which was being populated by
```

Comment:
.. so that when the secret content changes, we have a way to trigger a deployment.

Reply:
Reverted the comment and made it more clear why that environment variable is there.